src/photos.cs

```diff
@@ -1267,8 +1271,13 @@ interface PHProjectChangeRequest {
 		[Export ("projectExtensionData", ArgumentSemantic.Copy)]
 		NSData ProjectExtensionData { get; set; }
 
+		[Deprecated (PlatformName.MacOSX, 10, 14)]
```

Comment:
Any header comment on what to use instead?

Reply:
None :(